build/commands/lib/config.js

```diff
@@ -645,6 +644,66 @@ Config.prototype.buildArgs = function () {
   return args
 }
 
+Config.prototype.buildUpdaterArgsWin = function () {
+  // The following values come from our Omaha 3 implementation:
+  let argsFromOmaha3 = {
```

Comment:
I don't want to hard code these values in the scripts because it creates yet another dependency on them that prevents people from building without them. Couldn't we eliminate this and reduce patching by adding in a gni include at the end of branding.gni that overwrites all of them to our values?

Reply:
I am not sure it would work. I tried and got 
<pre>
import("//brave/updater/branding.gni")
  ^------------------------------------
This import contains "legacy_service_name_prefix"
See //brave/updater/branding.gni:7:30: defined here.
legacy_service_name_prefix = "brave"
                             ^------
Which would clobber the one in your current scope
See //chrome/updater/branding.gni:116:32: defined here.
  legacy_service_name_prefix = "cupdate"
                               ^--------
Executing import should not conflict with anything in the current
scope unless the values are identical.
</pre>

Reply:
@bridiver maybe something like this https://github.com/brave/brave-core/commit/3e558a05bc00962d83c7496d1442ff38b9520540, but I had to duplicate the values that we don't change.

Reply:
I think I found a nice solution: https://github.com/brave/brave-core/pull/19469/commits/15eb0edc4947c79ddd7c67e5126c111ae3c49d4a.